deployment/oss-vdb-test/k8s/cloudbuild.yaml

```diff
@@ -55,6 +56,17 @@ steps:
   args: ['push', 'gcr.io/$PROJECT_ID/combine-to-osv:$COMMIT_SHA']
 - name: gcr.io/cloud-builders/docker
   args: ['push', 'gcr.io/$PROJECT_ID/combine-to-osv']
+# Build/push debian converter images
+- name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/debian-first-version', '-t', 'gcr.io/$PROJECT_ID/debian-first-version:$COMMIT_SHA', '--build-arg', 'PROJECT=$PROJECT_ID', '.']
+  dir: 'osv.dev/vulnfeeds/tools/debian'
+- name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/debian-convert', '-t', 'gcr.io/$PROJECT_ID/debian-convert:$COMMIT_SHA', '--build-arg', 'PROJECT=$PROJECT_ID', '.']
+  dir: 'osv.dev/vulnfeeds/tools/debian'
+- name: gcr.io/cloud-builders/docker
+  args: ['push', 'gcr.io/$PROJECT_ID/debian-first-version']
+- name: gcr.io/cloud-builders/docker
+  args: ['push', 'gcr.io/$PROJECT_ID/debian-convert']
```

Comment:
Do you know if you also need to run `docker push gcr.io/$PROJECT_ID/[image]:$COMMIT_SHA`, or is the tag applied automatically?

Reply:
Ah I missed it, added now. I don't think it's tagged automatically